src/pages/index.js

```diff
@@ -0,0 +1,293 @@
+import {
+  enableValidation,
+  settings,
+  disableButton,
+  resetValidation,
+} from "../scripts/validation.js";
+import "./index.css";
+import "../vendor/fonts.css";
+import Api from "../utils/Api.js";
+import { showLoading } from "../utils/helpers.js";
+
+// Image file imports
+import logoImage from "../images/logo.svg";
+import avatarImage from "../images/avatar.jpg";
+import pencilImage from "../images/pencil.svg";
+import plusImage from "../images/plus.svg";
+import whitePencilIcon from "../images/White-Pencil.png";
+
+//
+let selectedCard;
+let selectedCardId;
+
+// Set initial image sources
+document.querySelector(".header__logo").src = logoImage;
+document.querySelector(".profile__avatar").src = avatarImage;
```

Comment:
`document.querySelector(".profile__avatar")`
`document.querySelector(".profile__name")`
`document.querySelector(".profile__description")`


It’s better to find **constant DOM elements** only 1 time at the top of the file so as not to waste resources on searching them again and again when you call a method (function) because searching is a very **hard** operation for the browser engine

**Please, correct it everywhere in the code**